lib/ansible/constants.py

```diff
@@ -104,7 +104,7 @@ def __getitem__(self, y):
 DEFAULT_SUBSET = None
 DEFAULT_SU_PASS = None
 # FIXME: expand to other plugins, but never doc fragments
-CONFIGURABLE_PLUGINS = ('become', 'cache', 'callback', 'cliconf', 'connection', 'httpapi', 'inventory', 'lookup', 'shell')
+CONFIGURABLE_PLUGINS = ('become', 'cache', 'callback', 'cliconf', 'connection', 'httpapi', 'inventory', 'lookup', 'shell', 'grpc')
```

Comment:
It doesn't look like `grpc` is actually configurable.

Reply:
ansible-doc CI test fails if the plugin is not added as a configurable plugin that's the reason it is added here and basic configuration document is added in plugin